src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/TestHost.cs

```diff
@@ -93,9 +136,14 @@ public byte[] GetPeerCert(bool exportAsPem)
             }
         }
 
-        public string GetFQDN()
+        public Stream Ping()
```

Comment:
We will likely not be keying off the return value, but rather off the HTTP response (i.e., we're likely to see a 50x if the server is misconfigured, a 200 if the server is set up right. 

I return a stream here because in all likelihood, the consumer of Ping is either a script (in which case, you care about the HTTP response code), or someone typing into a browser manually (in which case, a response where you can see "we work!" is useful)

An alternative is that we use `GetFqdn` as 'Ping`. However, having `Ping` is not super expensive and we can see very clearly the service's alive/not status. Thoughts, @roncain?